src/hmem_cuda.c

```diff
@@ -1,6 +1,6 @@
 /*
  * (C) Copyright 2020 Hewlett Packard Enterprise Development LP
- * (C) Copyright 2021-2022 Amazon.com, Inc. or its affiliates.
```

Comment:
we should just remove the date according to current guidance.

Reply:
I agree, though I'm curious if @shefty prefers the dates in the copyright notices. Otherwise we can start removing them as we update files.

Reply:
No - the dates aren't needed.  FWIW, Intel now has the same guidance.

Reply:
Removed year range in latest push